actors/test/commit_post_test.go

```diff
@@ -331,3 +333,307 @@ func TestCommitPoStFlow(t *testing.T) {
 		assert.True(t, networkStats.TotalPledgeCollateral.GreaterThan(big.Zero()))
 	})
 }
+
+func preCommitSectors(t *testing.T, v *vm.VM, batchSize int, worker, mAddr address.Address, sealProof abi.RegisteredSealProof) ([]abi.SectorNumber, []*miner.SectorPreCommitOnChainInfo) {
```

Comment:
Nit: push the helpers to the bottom of the file, so the test cases come first and detail emerges top-down.